pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -58,7 +58,7 @@ const (
 	// The minimum memory limit allocated to docker container: 150Mi
 	MinDockerMemoryLimit = 150 * 1024 * 1024
 
-	dockerProcessName     = "docker"
+	dockerProcessName     = "dockerd?"
```

Comment:
I checked the code and this seems to work, but it's very confusing because this is supposed to be the exact name. My suggestion would be either changing this to be a regex explicitly or make it a list of names ["dockerd", "docker"].